readme.md

```diff
@@ -48,27 +52,35 @@ async def run():
     sub = await nc.subscribe("foo", cb=message_handler)
 
     # Stop receiving after 2 messages.
-    await sub.unsubscribe(sid, limit=2)
+    await sub.unsubscribe(limit=2)
     await nc.publish("foo", b'Hello')
     await nc.publish("foo", b'World')
     await nc.publish("foo", b'!!!!!')
 
+    # Synchronous style with iterator also supported.
+    sub = await nc.subscribe("bar")
+    await nc.publish("bar", b'First')
+    await nc.publish("bar", b'Second')
+
+    try:
+        async for msg in sub.messages:
+            print(f"Received a message on '{msg.subject} {msg.reply}': {msg.data.decode()}")
+            await sub.unsubscribe()
+    except Exception as e:
+        pass
+
     async def help_request(msg):
-        subject = msg.subject
-        reply = msg.reply
-        data = msg.data.decode()
-        print("Received a message on '{subject} {reply}': {data}".format(
-            subject=subject, reply=reply, data=data))
-        await nc.publish(reply, b'I can help')
+        print(f"Received a message on '{msg.subject} {msg.reply}': {msg.data.decode()}")
+        await nc.publish(msg.reply, b'I can help')
 
     # Use queue named 'workers' for distributing requests
     # among subscribers.
     sub = await nc.subscribe("help", "workers", help_request)
 
     # Send a request and expect a single response
-    # and trigger timeout if not faster than 1 second.
+    # and trigger timeout if not faster than 500 ms.
     try:
-        response = await nc.request("help", b'help me', timeout=1)
+        response = await nc.request("help", b'help me', 0.5)
```

Comment:
nit: this should still be passed as a kwarg, as if the kwarg ordering in the function definition was changed this would stop working. Not a big deal.